src/utils/validators.js

```diff
@@ -51,13 +51,16 @@ class BaseValidator {
 export class UserValidator extends BaseValidator {
   static minUsernameLength = 3;
   static maxUsernameLength = 20;
-  static minPasswordLength = 3;
-  static maxPasswordLength = 20;
+  static minPasswordLength = 8;
+  static maxBioLength = 255;
+  static allowedThemes = ["light", "dark"];
 
-  static validateSignupInfo = (_email, _username, _password) => {
+  static validateSignupInfo = (_email, _username, _password, _bio, _theme) => {
     const email = this.validateString(_email, "email");
     const username = this.validateString(_username, "username");
     const password = this.validateString(_password, "password");
+    const bio = this.validateString(_bio, "bio");
+    const theme = this.validateString(_theme, "theme");
```

Comment:
We don't have an a bio and theme section in our signup form so that was why I left it out. Unless we're planning on adding those sections to form, I don't think this is necessary.

Reply:
So we just remove this code?

Reply:
Yeah unless you're planning on adding the inputs in the handlebars and making sure it works.